### PR TITLE
New HomePage design

### PR DESCRIPTION
**Pull request by mvaivre**
Brand new design, following [this mockup](https://www.figma.com/file/7SiG11k6DJiSRvysd2Kadw/Alephium-Explorer?node-id=537%3A582). 

To keep this small enough, I'll take care of tasks below in a following PR:

- [ ] Add CardWithCharts
- [ ] Add DetailedChard in a modal, when clicking on a CardWithChart
- [ ] Misc (matching logo, page switch next to table header)

### CODE REVIEW

File: src/components/AppFooter.tsx [hunk 1]
```diff
@@ -80,9 +80,8 @@ const socialMediaData = [
 export default styled(AppFooter)`
   display: flex;
   justify-content: space-between;
-  background-color: ${({ theme }) => theme.bgTertiary};
+  backdrop-filter: blur(20px);
```
[the remark this answers is not in the document]
Comment (nop33) [reply src/components/AppFooter.tsx:83]: oh, and the waves are not blurred out behind the boxes

**Comment by nop33** [comment]
Is this the desired effect on the footer? (chrome/safari, iPhone 13 Pro Max)

image

File: src/components/AppFooter.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvaivre) [reply src/components/AppFooter.tsx:83]: Yeah I know that Firefox doesn't support the background blur, that's sad. Don't understand why they wouldn't do it.

File: src/components/SearchBar.tsx [hunk 2]
```diff
@@ -94,7 +95,7 @@ const SearchBar = ({ className }: SearchBarProps) => {
   }
 
   return (
-    <div className={className}>
+    <motion.div className={className} layoutId="searchBar">
```
[the remark this answers is not in the document]
Comment (mvaivre) [reply src/components/SearchBar.tsx:98]: It's for the magic animation ✨, moving the search bar from one place to another (main content to AppHeader here) :)

**Comment by mvaivre** [comment]
> Is this the desired effect on the footer? (chrome/safari, iPhone 13 Pro Max)

For now yes. But I may eventually decide to tweak the footer more (reduce the brightness) or entirely remove the background filter on mobile.

File: src/components/AppFooter.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (LeeAlephium) [reply src/components/AppFooter.tsx:83]: Could simply be not enough programmer hours put into it

[the remark this answers is not in the document]
Comment (nop33) [reply src/components/AppFooter.tsx:83]: Maybe we can use https://developer.mozilla.org/en-US/docs/Web/CSS/@supports to see if this property is supported and provide a different style for firefox.

**Comment by mvaivre** [comment]
Superseded by #98

**Pull request closed without merging**